scripts/generate_dump

```diff
@@ -859,22 +902,26 @@ save_log_files() {
 ###############################################################################
 # Save warmboot files
 # Globals:
-#  TARDIR, TARFILE, TAR, DUMPDIR, TECHSUPPORT_TIME_INFO
+#  TARDIR, TARFILE, TAR, DUMPDIR, TECHSUPPORT_TIME_INFO, NOOP
 # Arguments:
 #  None
 # Returns:
 #  None
 ###############################################################################
 save_warmboot_files() {
     # Copy the warmboot files
-    mkdir -p $TARDIR
-    $CP $V -rf /host/warmboot $TARDIR
-
-    start_t=$(date +%s%3N)
-    ($TAR $V --warning=no-file-removed  -rhf $TARFILE -C $DUMPDIR --mode=+rw \
-        $BASE/warmboot \
-        || abort "${ERROR_TAR_FAILED}" "Tar append operation failed. Aborting for safety.") \
-        && $RM $V -rf $TARDIR
+    if $NOOP; then
+        echo "$CP $V -rf /host/warmboot $TARDIR"
+    else
+        mkdir -p $TARDIR
+        $CP $V -rf /host/warmboot $TARDIR
+
+        start_t=$(date +%s%3N)
```

Comment:
This line needs to be out of the condition to be used for calculating the time.

Reply:
Another question is if we need to log the time of the output generation for noop case. What do you think?

Reply:
It's not needed, adding it to be consistent with other commands.

Reply:
> This line needs to be out of the condition to be used for calculating the time.

:grimacing: